packager/packager.js

```diff
@@ -34,6 +34,10 @@ var parseCommandLine = require('./parseCommandLine.js');
 var webSocketProxy = require('./webSocketProxy.js');
 
 var options = parseCommandLine([{
+  command: 'origin',
+  default: 'http://localhost',
+  type: 'string',
+}, {
```

Comment:
we could split up protocol and hostname since most of the time (for development) the protocol is http and only the hostname needs to change.